forge-game/src/main/java/forge/game/combat/GlobalAttackRestrictions.java

```diff
@@ -133,7 +134,7 @@ public static GlobalAttackRestrictions getGlobalRestrictions(final Player attack
         }
 
         for (final GameEntity defender : possibleDefenders) {
-            final int defMax = getMaxAttackTo(defender);
+            final int defMax = StaticAbilityAttackRestrict.attackRestrictNum(defender);
```

Comment:
The KW and the global rule above there should be mergeable too?

Reply:
Worth a shot!

Reply:
✅